catalog/core/catalog-core-backupplugin/src/main/java/ddf/catalog/backup/CatalogBackupPlugin.java

```diff
@@ -55,6 +57,10 @@
 
     private static final String TEMP_FILE_EXTENSION = ".tmp";
 
+    private static final ExecutorService EXECUTOR_SERVICE =
+            Executors.newFixedThreadPool(Runtime.getRuntime()
```

Comment:
Single thread is probably enough here but if it wasn't then you should use the system property for thread pool size.

https://github.com/codice/ddf/blob/ddf-2.8.2/distribution/ddf-common/src/main/resources/etc/system.properties#L51